README.md

```diff
@@ -72,6 +72,28 @@ Cookies.remove('name', { path: '/' }); // => true
 
 *Note: when deleting a cookie, you must pass the exact same path, domain and secure options that were used to set the cookie, unless you're relying on the default options that is.*
 
+## JSON
+
+js-cookie provides automatic JSON storage for cookies.
+
+When creating a cookie you can pass an Array or Object Literal instead of a string in the value. If you do so, js-cookie store the string representation of the object according to the `JSON.stringify` api (if available):
```

Comment:
Strictly technical, it doesn't need to be an Array/Object literal, does it? E.g.
```js
Cookies.set("foo", new Array("foo"))

In that case I'd rather write something like "...you can pass an instance of Array or Object..."